bee-hive/tests/bee/workflow.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: beehive/v1alpha1
+kind: Workflow
+metadata:
+  name: beetest-deployment
+  labels:
+    app: testapp
+spec:
+  strategy:
+    type: sequence
+    output: nonal
+  template:
+    metadata:
+      labels:
+        app: testapp
+    agents:
+      - name: beetest
+    prompt: Welcome
+    start: begin
+    steps:
+      - name: begin
+        agent: crewtest
+        dependencies:
```

Comment:
The `dependencies` is not in the schema for now.

Reply:
removed from both tests